frontend/src/scenes/billing/Billing.tsx

```diff
@@ -4,17 +4,17 @@ import { CurrentPlan } from './CurrentPlan'
 import { CurrentUsage } from './CurrentUsage'
 import { BillingEnrollment } from './BillingEnrollment'
 import { useValues } from 'kea'
-import { userLogic } from 'scenes/userLogic'
 import './Billing.scss'
+import { billingLogic } from './billingLogic'
 
 export function Billing(): JSX.Element {
-    const { user } = useValues(userLogic)
+    const { billing } = useValues(billingLogic)
 
     return (
         <div className="billing-page">
-            <PageHeader title="Billing &amp; usage information" />
+            <PageHeader title="Billing &amp; usage information iajsdfoijsadfioasjdifojas" />
```

Comment:
`iajsdfoijsadfioasjdifojas` --- is this a test to make sure the reviewer goes through all the files? :D

Reply:
hahaha 🙈 ... it was just a test